fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -29,17 +32,35 @@ class ConditionalSelectionRate(ClassificationMoment):
 
     """
 
+    def __init__(self, ratio=1.0):
+        """Initialize with the ratio value."""
+        super(ConditionalSelectionRate, self).__init__()
+        if not (0 < ratio <= 1):
+            raise ValueError(_MESSAGE_RATIO_NOT_IN_RANGE)
+        self.ratio = ratio
+
     def default_objective(self):
         """Return the default objective for moments of this kind."""
         return ErrorRate()
 
-    def load_data(self, X, y, event=None, **kwargs):
+    def load_data(self, X, y, event=None, utilities=None, **kwargs):
         """Load the specified data into this object.
 
         This adds a column `event` to the `tags` field.
+
+        The `utilities` is a 2-d array which correspond to g(X,A,Y,h(X)) as mentioned in the paper
```

Comment:
This is not quite right. The array `utilities` should have the utilities in columns not in rows--similarly to how `X` is sharped and how multi-class labels `y` and `predict_proba` are treated in `sklearn`.